OpenRA.Mods.Common/Activities/Move/AttackMoveActivity.cs

```diff
@@ -21,9 +21,12 @@ public class AttackMoveActivity : Activity
 	{
 		readonly Func<Activity> getInner;
 		readonly bool isAssaultMove;
-		AutoTarget autoTarget;
-		AttackMove attackMove;
+		readonly AutoTarget autoTarget;
+		readonly AttackMove attackMove;
+
+		bool runningInnerActivity = false;
```

Comment:
I spent the first few minutes looking at this confused because I didn't understand that the "inner" activity here refers to the move, and is not just a misnamed label for the child activity. Can we rename this and `getInner` to specifically mention "move"?

Reply:
It isn't always a `Move` activity. It can also be `Follow`, `Fly` or `FlyFollow` and potentially other things as well.

Reply:
Those are still move (little m) activities!